modules/mobile/app/models/mobile/v0/appointment.rb

```diff
@@ -68,14 +68,14 @@ class Appointment < Common::Resource
       attribute :best_time_to_call, Types::Array.optional
       attribute :friendly_location_name, Types::String.optional
 
-      def self.toggle_non_prod_id!(id)
+      def self.convert_non_prod_id!(id)
         return id if Settings.hostname == 'api.va.gov' || id.nil?
```

Comment:
Is this hostname check essentially the same as "if rails.env.production?" If so, I think we should change it to make it clearer that this is really just "don't do this in prod".

Reply:
I'm assuming they're the same check but I'm hesitant to change it on the off chance there's some obscure edge case that doesn't make them the same. I don't know enough about the devops of vets-api to say for certain.

Reply:
Fair.